### ISSUE TITLE
Proxy Forward

### ISSUE THREAD

**Issue by hamedsbt** [evilsocket/opensnitch#738]
Many applications on Linux don't support the proxy option, such as Skype. Therefore, an option to forward its request to a specified proxy server is necessary.

Thank you!

**Comment by gustavo-iniguez-goya** [comment]
Hi @hamedsbt ,

This feature was already requested here #437 . I wrote a PoC some months ago, but it was a wasted of time, because we need to place a mark on every packet that the application sends. For UDP it could work, because we intercept `udp_sendmsg` syscall. But for TCP we need to intercept tcp_sendmsg/tcp_recvmsg in order to mark every packet, because right now we only intercept new connections.

On the other hand... I also wrote a PoC to intercept tcp_sendmsg/recvmsg to get statistics about amount of bytes sent by the applications. SoI have a  general idea on how this could work, and once one or the other feature is finsihed, it'll be easier to add the other one, but I need to glue all the pieces together, test it, and see if it works as expected.
